docs/04-reference/wingsdk-api.md

```diff
@@ -5581,7 +5581,7 @@ data to be inserted.
 ##### `list` <a name="list" id="@winglang/sdk.cloud.ITableClient.list"></a>
 
 ```wing
-list(): any
+list(): MutArray<Json>
```

Comment:
Why not `Array<Json>`?

Reply:
I have no clue why was this generated like this, do you?

Reply:
@staycoolcall911 , do you know why it created a MutArray doc?

Reply:
https://github.com/winglang/wing/blob/main/apps/jsii-docgen/src/docgen/transpile/wing.ts#L82

Reply:
@MarkMcCulloh , can you explain it to me as If I was from product? 🤷

Reply:
The jsii documentation generator doesn't currently have the concept of a mutable/immutable array, just an array. I figured that MutArray is technically the safer option to default to since that's what a js array is.

Reply:
yeah, I think it make sense from a safety and correctness.

This is something we need to think about @eladb , but it is not in the context of this PR.

opened https://github.com/winglang/wing/issues/2335

Reply:
It's a 1 liner fix, so it's best if we can make a decision, but of course it's not a blocker for this PR

Reply:
In the code right now, it's return [Array](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array), not our array.

Reply:
I see, so can we return our array?

Reply:
Our ImmutableArray is not generic, so I don't know if this is possible.

@Chriscbr ?

Reply:
Lets move this discussion to the other issue, then (#2335) 
Closing this one, @eladb I need your reviewer approval